flaxandteal/coral-arches
# Fix/#2264 location data

### Description
Update the location details so that they are all unified

### Tests
- reload the following models:
    - Monument
    - Monument Revision
    - Consultation
    - Activity
- coral reload
- make webpack 

Check each workflow to ensure the location details page contains:
- Building name
- Street
- Town or City
- County
- Postcode
- Townland
- Council
- Area Type
- Area Name

workflows containing location details are:
- Evaluation Meeting
- Designation
- Excavation Licensing
- Planning Consultation
- Add Building
- Add IHR
- Add Monument
- Add Garden
- Incident Report

- check that the designation workflow step states 'Location Details'
- Check that the help icon displays info for the listed nodes
